tests/cypress/support/commands.ts

```diff
@@ -551,10 +551,32 @@ Cypress.Commands.add('removeClusterLabels', (clusterName, key, value) => {
   cy.contains('.title', 'Clusters').should('be.visible');
   cy.filterInSearchBox(clusterName);
   cy.open3dotsMenu(clusterName, 'Edit Config');
-  cy.get('div[class="row"] div[class="key-value"] button.role-link').first().click();
+  // TODO: Remove below label remove logic after 
+  // issue: # https://github.com/rancher/dashboard/issues/9563
+  // Below code will work for 3 labels on the cluster, 2 with disabled labels,
+  // 1 with actual removable label.
+  cy.get('body').then((body) => {
+    if (body.find('span[class="switch hand"]')) {
+      cy.get('span[name="label-system-toggle"]').click();
+      let disabled_button = cy.get('div[class="row"] div[class="key-value"] button.role-link')
+
+      if (disabled_button < 3) {
+        cy.log("working");
```

Comment:
Maybe leftover?
Probably better something like "Only 2 default label. Nothing to do"

Reply:
Oops, I just saw the comment but I updated with different log message. I will update it with your suggested changes in next patch set.